microsoft-edge/devtools-guide-chromium/contact.md

```diff
@@ -6,39 +6,40 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: devtools
-ms.date: 09/29/2021
+ms.date: 03/07/2022
 ---
 # Contact the Microsoft Edge DevTools team
 
-Send your feedback to help improve Microsoft Edge DevTools.  Contact the Microsoft Edge DevTools team to submit feature requests or bug reports of issues about developing for Microsoft Edge.
+Send your feedback to help improve Microsoft Edge DevTools.  Contact the Microsoft Edge DevTools team to submit feature requests or bug reports, or ask questions about Microsoft Edge DevTools.
 
-There are several ways to get in touch with the Microsoft Edge DevTools team.
+There are several ways to get in touch with the Microsoft Edge DevTools team:
+
+* Using the **Send feedback** window to quickly report bugs while using DevTools.
+* Using the **DevTools** repo to submit feature requests, questions, and have longer conversations with the team.
+* Using **The Web We Want** initiative to submit cross-browser suggestions.
```

Comment:
```suggestion
* Using the **Web We Want** initiative to submit cross-browser suggestions.
```

Reply:
And maybe all "ing" such as "using" to "use" and "clicking" to "click" ...

Reply:
Or, as suggested by Michael, start with the goal such as "To submit cross-browser suggestions, use the **Web We Want** initiative." and "To quickly report bugs from DevTools, use the **Sendfeedback** window."

Reply:
Sounds great. Will address in a new commit.